test/e2e/testing-manifests/kubectl/busybox-cronjob.yaml.in

```diff
@@ -15,7 +15,7 @@ spec:
         spec:
           containers:
           - name: test
-            image: busybox
+            image: {{.BusyboxImage}}
```

Comment:
i think these need to be `BusyBoxImage` because the parser is case sensitive:
https://github.com/kubernetes/kubernetes/blob/3f585835d0a38462bb928454245ec01e834e53a0/test/e2e/common/util.go#L78

Reply:
thanks, was warming up my test env separately